charts/quine-enterprise/values.yaml

```diff
@@ -106,6 +106,10 @@ metrics:
     host: ""
     port: 8086
 
+# Enable Basic Authentication with NGINX
```

Comment:
It could be nice to include a link here to the nginx documentation of the format we expect in the `htpasswd` value.

Reply:
I've added some documentation on the expected format of this (brilliant idea 🧠), and I also plan on providing example `yaml` in our enterprise docs after this PR is merged to serve as a reference on how to enable basic auth using our Helm charts.

Reply:
It looks great!